.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -1,3 +1,17 @@
+
+---
+name: S2N Issue
+about: Template
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Security issue notifications
+If you discover a potential security issue in s2n we ask that you notify
+AWS Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public github issue.
+
 ### Problem:
 
 A short description of what the problem is and why we need to fix it. Add reproduction steps if necessary.
```

Comment:
While you're updating this, could you also add newlines after each section? It's annoying to replace the text when the next title is on the next line.